content/intermediate/operations/resource-importing/index.md

```diff
@@ -323,16 +571,42 @@ Choose to follow cleanup steps shown next to clean up resources you created with
 
 1. Make sure you are in the directory: `code/workspace/resource-importing`
 2. Update the `resource-importing.yaml` template file to remove the `DeletionPolicy: Retain` line from the `SNSTopic2` resource definition, and save the template.
```

Comment:
Don't we want to set `DeletionPolicy: Delete` so the resources are cleaned up?

Reply:
We are removing the Deletion Policy section in the template so that the resources are cleaned up upon deletion

Reply:
If I remove `DeletionPolicy` I'm unable to update the stack - "no changes detected" and this is how I got around it, by explicitly setting `Delete`

Reply:
I have changed the command, earlier the errors were on the change sets, now that we are directly updating stacks it should be sufficient. I have tested this on the cloud 9 console